recipes/meshplex/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "meshplex" %}
+{% set version = "0.13.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/nschloe/meshplex/archive/v{{ version }}.tar.gz
+  sha256: ceadd26da8c4bfdd4e543f48944b43a5c38ad4faa55563330651fb6d6f6ac765
+
+build:
+  number: 0
+  skip: true  # [py==38]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - importlib_metadata
+    - pip
+    - python
+    - numpy >=1.9
+    - scipy
+    - meshio >=4,<5
+
+  run:
+    - importlib_metadata
+    - python
+    - numpy >=1.9
+    - scipy
+    - meshio >=4,<5
+
+test:
+  imports:
+    - meshplex
+
+about:
+  home: "https://github.com/nschloe/meshplex"
+  license: GPL-3.0-only
```

Comment:
Looks like it is "or-later." See https://github.com/nschloe/meshplex/blob/master/setup.cfg#L13

Reply:
Yes, you are right